POC/ColorPicker/ColorPicker/ColorPickingFunctionality/SystemEvents/MouseEvent.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace ColorPicker.ColorPickingFunctionality.SystemEvents
+{
+    class MouseEvent : SystemHook
```

Comment:
Should this be named something like RegisteredMouseEventHook? To clearly show the intent of the class? (It doesn't represent a mouse event, but rather implements the capturing and dispatching of the event to the callback.)